packages/gatsby-recipes/src/providers/npm/package.test.js

```diff
@@ -8,15 +8,15 @@ const resourceTestHelper = require(`../resource-test-helper`)
 
 const root = path.join(os.tmpdir(), uuid.v4())
 fs.mkdirSync(root)
-const pkgResource = { name: `glob` }
+const pkgResource = { name: `bar` }
 
 test(`plan returns a description`, async () => {
   const result = await pkg.plan({ root }, pkgResource)
 
-  expect(result.describe).toEqual(expect.stringContaining(`Install glob`))
+  expect(result.describe).toEqual(expect.stringContaining(`Install bar`))
```

Comment:
maybe we should go for a small package like https://github.com/developit/dlv. A zero dependency and small module. Feel free to pick anything else, maybe we should publish a noop package to @gatsbyjs

Reply:
haha yeah — I enjoyed the names mostly. Switched.